code/game/verbs/who.dm

```diff
@@ -36,7 +36,7 @@
 
 	var/list/Lines = list()
 
-	if(holder && (R_ADMIN & holder.rights || R_MOD & holder.rights))
+	if((holder?.rights & (R_ADMIN|R_MOD)) || ((src?.mob?.stat == DEAD) && (holder?.rights & (R_MENTOR))))
```

Comment:
This shouldn't cause issues, because if there is no `holder` it will try to do a bitflag operation of `null & (R_ADMIN|R_MOD)`, which should be valid, but kind of not what you were intending. Be careful on how you use `.?`

This, on the other hand, will cause a runtime error: `src?.mob?.stat == DEAD`
As it will try to associate the DEAD constant to a null var.